openapi.yaml

```diff
@@ -188,44 +224,64 @@ components:
             $ref: '#/components/schemas/Intersection'
         rotary_name:
           type: string
+          description: The name of the traffic circle.
         rotary_pronunciation:
           type: string
+          description: >-
+            Pronunciation of the rotary name (if available).
+            The format of this varies by implementation/vendor.
         driving_side:
           type: string
           enum:
             - left
             - right
+          description: The side of the road on which driving is legal for this step.
         voiceInstructions:
           type: array
           items:
             $ref: '#/components/schemas/VoiceInstruction'
+          description: >-
+            A list of announcements which should be spoken at various points along the maneuver.
+            This is a Valhalla extension to the OSRM spec.
         bannerInstructions:
           type: array
           items:
             $ref: '#/components/schemas/BannerInstruction'
+          description: >-
+            A list of announcements which should be displayed prominently on screen at various points along the maneuver.
+            This is a Valhalla extension to the OSRM spec.
     Annotation:
       type: object
       properties:
+        # TODO: Refactor into properties?
```

Comment:
Might make sense to refactor some of these out to top level objects and/or add another property (I think `title`) for model naming control, but I'll see how badly it screws up the generated models first.